### PR TITLE
security_group_info module added which is missing in openstack.cloud.* collection

### PR DESCRIPTION
**Pull request by tischrei**

### CODE REVIEW

**Review by Polina-Gubina — APPROVED** [review]

File: plugins/modules/security_group_info.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCUMENTATION = '''
+module: security_group_info
+short_description: Lists security groups
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.0.1"
```
Comment (gtema) [inline plugins/modules/security_group_info.py:18]: version too old

File: plugins/modules/security_group_info.py [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+options:
+  description:
+    description:
+      - Description of the security group
+    type: str
+  project_id:
+    description:
+      - Specifies the project id as filter criteria
+    type: str
+  security_group:
```
Comment (gtema) [inline plugins/modules/security_group_info.py:31]: best practice in openstack collection, which we follow, is to have attribute `name` as name or id

File: plugins/modules/security_group_info.py [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+        security_group=dict(required=False),
+        project_id=dict(required=False)
+    )
+
+    def run(self):
+
+        data = []
+        query = {}
+
+        if self.params['security_group']:
```
Comment (gtema) [inline plugins/modules/security_group_info.py:127]: please compare with https://github.com/opentelekomcloud/ansible-collection-cloud/blob/master/plugins/modules/waf_domain_info.py for name/id handling

**Review by gtema — CHANGES_REQUESTED** [review]

File: plugins/modules/security_group_info.py (hunk 1 above)
Comment (tischrei) [reply plugins/modules/security_group_info.py:18]: done

**Comment by tischrei** [comment]
review changes finished.

**Comment by tischrei** [comment]
recheck

**Comment by tischrei** [comment]
pls recheck @gtema

**Review by gtema — APPROVED** [review]

**Pull request merged**